ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuiteRunner.scala

```diff
@@ -136,7 +132,6 @@ final class LedgerTestSuiteRunner(
       .onComplete { result =>
         participantSessionManager.closeAll()
         completionCallback(result)
-        executionContext.shutdown()
```

Comment:
I'm late in noticing this, but can you please dispose of the actor system when it's no longer needed? It's practically never needed, but it's just to make sure the runner can be potentially used as a library without any leak.

Reply:
Done.